CHANGELOG.md

```diff
@@ -0,0 +1,69 @@
+# Changelog
+
+### 0.9.1 (2015-07-20)
+
+* make default `controllerWhitelist` more specific
+
+---
+### 0.9.0 (2015-07-09)
+
+* add support for controller/action specific middleware
```

Comment:
Can't link to issues as many of these didn't have one. Could link to the PRs or to the commits.